source/crud/tests/v2/bulkWrite-arrayFilters.yml

```diff
@@ -65,3 +65,60 @@ tests:
         data:
           - {_id: 1, y: [{b: 2}, {b: 2}]}
           - {_id: 2, y: [{b: 0}, {b: 2}]}
+  -
+    description: "BulkWrite with arrayFilters using update pipelines"
```

Comment:
This test has to be skipped on MongoDB < 4.2.

Reply:
Please move this to a new test file and use `minServerVersion`. See https://github.com/mongodb/specifications/blob/master/source/crud/tests/v2/bulkWrite-arrayFilters.yml for prior art.

I'd also suggest using the exact server release in the corresponding SERVER ticket's fixVersion. Using a specific version (rather than 4.2.0) will make it easier for drivers to automatically start running this test when the feature first becomes available in the "latest" server binary.

Reply:
I have added a new test file - `updateWithPipelines.yml`.